web-marketplace/src/components/organisms/BasicInfoForm/BasicInfoForm.tsx

```diff
@@ -54,6 +56,7 @@ const BasicInfoForm: React.FC<BasicInfoFormProps> = ({
   });
   const setErrorBannerTextAtom = useSetAtom(errorBannerTextAtom);
   const { confirmSave, isEdit, isDirtyRef } = useProjectEditContext();
+  const { formRef, shouldNavigateRef } = useCreateProjectContext();
```

Comment:
I'm getting this error if trying to edit an on-chain project: `useCreateProjectContext must be used within a nested ProjectCreate route`

Reply:
I missed that, thanks. This is fixed.